frontend/static/style.css

```diff
@@ -101,8 +101,36 @@ body {
     width:16px;
     height:16px;
     cursor:pointer;
+    position: relative;
 }
 
+.module-window-terminal .terminal-label {
+    position: absolute;
+    font-variant: small-caps;
+    font-size: 11px;
+    top: 5px;
+    background-color: white;
+    display: none;
```

Comment:
with this line commented out, so that you can see what hover states _would_ look like, we have:

![image](https://user-images.githubusercontent.com/2560/78882247-29524b00-7a9b-11ea-9ca1-5b2d4be89810.png)